pilz_control/test/unittest_pilz_joint_trajectory_controller.cpp

```diff
@@ -355,6 +419,40 @@ TEST_F(PilzJointTrajectoryControllerTest, testGoalCancellingDuringHold)
   EXPECT_TRUE(isControllerInHoldMode());
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////////
+//    Testing the correct handling of trajectories that violate the acceleration limits    //
+/////////////////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * @brief Send a trajectory that has too high acceleration and make sure controller does not execute it.
+ */
+TEST_F(PilzJointTrajectoryControllerTest, testTrajectoryWithTooHighAcceleration)
+{
+  ASSERT_TRUE(performFullControllerStartup(&robot_driver_));
+
+  // unhold controller and record start pose;
+  std_srvs::TriggerRequest req;
+  std_srvs::TriggerResponse resp;
+  EXPECT_TRUE(manager_->triggerUnHold(req, resp));
+  EXPECT_TRUE(resp.success);
+  EXPECT_TRUE(isControllerInUnholdMode());
+
+  // Make sure, robot moves for slow motion
+  GoalType goal{ generateSimpleGoal<RobotDriver>(&robot_driver_, ros::Duration(DEFAULT_GOAL_DURATION_SEC)) };
+  action_client_.sendGoal(goal);
+  EXPECT_TRUE(manager_->controller_->is_executing());
+  action_client_.waitForActionResult();
+  EXPECT_TRUE(updateUntilNoRobotMotion<RobotDriver>(&robot_driver_));
+
+  // Now sending a quicker motion which should trigger the acceleration limit and not move the robot
+  goal = generateSimpleGoal<RobotDriver>(&robot_driver_, ros::Duration(DEFAULT_GOAL_DURATION_SEC), 1E4);
+  EXPECT_TRUE(updateUntilNoRobotMotion<RobotDriver>(&robot_driver_));
```

Comment:
You already checked this two lines above, so why do you check it again?

Reply:
Done.